pkgs/tools/virtualization/reg/default.nix

```diff
@@ -14,6 +16,14 @@ buildGoModule rec {
     sha256 = "1jlza1czfssssi3y9zi6kr8k9msfa7vp215ibhwbz4h97av5xw5m";
   };
 
+  patches = lib.optionals (stdenv.hostPlatform.system == "aarch64-darwin") [
```

Comment:
Let's apply the patch unconditionally, since that will be the case if the PR is merged.